### ISSUE TITLE
Align CSS-doesn't-load wiki page to new version of css-loader

### ISSUE THREAD

**Issue by rmja** [aurelia/webpack-plugin#168]
**I'm submitting a bug report**

* **Library Version:**
4.0.0

**Please tell us about your environment:**
* **Operating System:**
all

* **Node Version:**
all
* **NPM Version:**
all
* **JSPM OR Webpack AND Version**
all

* **Browser:**
all
* **Language:**
all

**Current behavior:**
[The wiki page describing how to resolve the issue when css files are included directly from the template](https://github.com/aurelia/webpack-plugin/wiki/CSS-doesn't-load) is not correct for the recently released `css-loader` version 4. [The new version has changed the default of `esModule` from `false` to `true`](https://github.com/webpack-contrib/css-loader/releases/tag/v4.0.0) breaking the included css.
The sample code should instead be:

```js
module: {
  rules: [
    {
      test: /\.css$/i,
      issuer: /\.html?$/i,
      use: [{ loader: "css-loader", options: { esModule: false }}]
    },
    {
      test: /\.css$/i,
      issuer: /\.[tj]s$/i,
      use: ["style-loader", "css-loader"]
    }
  ]
}
```

**Expected/desired behavior:**
The documentation should align with the newest version of `css-loader`.

(I cannot figure out how to create a PR for the wiki)

**Comment by elitastic** [comment]
Many thanks for this info,

I also had this error and it made me crazy the last 4-5 hours till I found your post.

**Comment by 3cp** [comment]
Note the above loader-webpack PR will fix this, requiring no user code/config change.

**Comment by 3cp** [comment]
Fix in aurelia-loader-webpack 2.2.2 is released. @jods4 this can be closed.
